packages/@lwc/module-resolver/tsconfig.json

```diff
@@ -6,5 +6,6 @@
         "declarationDir": "dist/types"
     },
 
-    "include": ["src/"]
+    "include": ["src/"],
+    "exclude": ["**/__tests__"]
```

Comment:
Some upgrade to Jest caused it to start complaining about `__test__` files in the `dist/` directory. Fixed it so we are no longer putting `__test__` files in `dist/`.